dvc/utils/compat.py

```diff
@@ -18,6 +18,11 @@
 # simplified version of ipython_genutils/encoding.py
 DEFAULT_ENCODING = sys.getdefaultencoding()
 
+if _ver[:2] < (3, 5):
+    RecursionError = RuntimeError
```

Comment:
really don't like this but it was the simplest way I could expose this for external